<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Chan.local" name="NexiaDriver" tests="4" failures="1" timestamp="23 Jun 2016 04:12:42 GMT" time="47305.637" errors="0">
  <testcase name="@AfterMethod tearDown" time="0.0" classname="com.projectname.test.functional.clinicalsettings.practiceLibrary.EncounterEndToEndSix">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Error communicating with the remote browser. It may have died.
Build info: version: &amp;apos;2.43.1&amp;apos;, revision: &amp;apos;5163bceef1bc36d43f3dc0b83c88998168a363a0&amp;apos;, time: &amp;apos;2014-09-10 09:43:55&amp;apos;
System info: host: &amp;apos;Chan.local&amp;apos;, ip: &amp;apos;192.168.56.169&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.10.2&amp;apos;, java.version: &amp;apos;1.8.0_73&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.43.1', revision: '5163bceef1bc36d43f3dc0b83c88998168a363a0', time: '2014-09-10 09:43:55'
System info: host: 'Chan.local', ip: '192.168.56.169', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.10.2', java.version: '1.8.0_73'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:593)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:614)
	at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:468)
	at com.projectname.testutils.baseclass.TestBaseClass.tearDown(TestBaseClass.java:351)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7056 [/127.0.0.1] failed: Connection refused
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:140)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:318)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:363)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:219)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:215)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:165)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:362)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:572)
	... 18 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:72)
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:123)
	... 32 more
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@BeforeMethod genericSetUp" time="0.0" classname="com.projectname.test.functional.clinicalsettings.practiceLibrary.VerifyCancelEncounter">
    <skipped/>
  </testcase> <!-- @BeforeMethod genericSetUp -->
  <testcase name="@BeforeMethod genericSetUp" time="0.0" classname="com.projectname.test.functional.clinicalsettings.practiceLibrary.Verifyremoveandpresentreusewidget">
    <skipped/>
  </testcase> <!-- @BeforeMethod genericSetUp -->
  <testcase name="@BeforeMethod genericSetUp" time="0.0" classname="com.projectname.test.functional.clinicalsettings.practiceLibrary.CoSignEncounter">
    <skipped/>
  </testcase> <!-- @BeforeMethod genericSetUp -->
  <testcase name="verifyEncounterEndToEndSix" time="47301.254" classname="com.projectname.test.functional.clinicalsettings.practiceLibrary.EncounterEndToEndSix">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Error communicating with the remote browser. It may have died.
Build info: version: &amp;apos;2.43.1&amp;apos;, revision: &amp;apos;5163bceef1bc36d43f3dc0b83c88998168a363a0&amp;apos;, time: &amp;apos;2014-09-10 09:43:55&amp;apos;
System info: host: &amp;apos;Chan.local&amp;apos;, ip: &amp;apos;192.168.56.169&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.10.2&amp;apos;, java.version: &amp;apos;1.8.0_73&amp;apos;
Driver info: driver.version: FirefoxDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '2.43.1', revision: '5163bceef1bc36d43f3dc0b83c88998168a363a0', time: '2014-09-10 09:43:55'
System info: host: 'Chan.local', ip: '192.168.56.169', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.10.2', java.version: '1.8.0_73'
Driver info: driver.version: FirefoxDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:593)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:614)
	at org.openqa.selenium.firefox.FirefoxDriver.getScreenshotAs(FirefoxDriver.java:323)
	at com.projectname.testutils.baseclass.TestBaseClass.assertTrue(TestBaseClass.java:429)
	at com.projectname.test.functional.clinicalsettings.practiceLibrary.EncounterEndToEndSix.encounterTemplate(EncounterEndToEndSix.java:164)
	at com.projectname.test.functional.clinicalsettings.practiceLibrary.EncounterEndToEndSix.verifyEncounterEndToEndSix(EncounterEndToEndSix.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.http.conn.HttpHostConnectException: Connect to 127.0.0.1:7056 [/127.0.0.1] failed: Connection refused
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:140)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:318)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:363)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:219)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
	at org.openqa.selenium.remote.HttpCommandExecutor.fallBackExecute(HttpCommandExecutor.java:215)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.execute(NewProfileExtensionConnection.java:165)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.execute(FirefoxDriver.java:362)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:572)
	... 18 more
Caused by: java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:72)
	at org.apache.http.impl.conn.HttpClientConnectionOperator.connect(HttpClientConnectionOperator.java:123)
	... 32 more
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- verifyEncounterEndToEndSix -->
  <testcase name="verifyCancelEncounter" time="0.001" classname="com.projectname.test.functional.clinicalsettings.practiceLibrary.VerifyCancelEncounter">
    <skipped/>
  </testcase> <!-- verifyCancelEncounter -->
  <testcase name="verifyremoveandpresentreusewidgets" time="0.004" classname="com.projectname.test.functional.clinicalsettings.practiceLibrary.Verifyremoveandpresentreusewidget">
    <skipped/>
  </testcase> <!-- verifyremoveandpresentreusewidgets -->
  <testcase name="verifyCoSignEncounter" time="0.001" classname="com.projectname.test.functional.clinicalsettings.practiceLibrary.CoSignEncounter">
    <skipped/>
  </testcase> <!-- verifyCoSignEncounter -->
</testsuite> <!-- NexiaDriver -->
